tests/scripts/actionutils.sh

```diff
@@ -445,7 +450,7 @@ function upgrade_multinode() {
         done
         res=$( ( lxc exec $container -- sh -c "microceph.ceph osd status" | fgrep -c "exists,up" ) )
         if [[ $res -ne $expect ]] ; then
-            echo "Expected $expect OSD up, got $res"
```

Comment:
please refrain from modifying unrelated lines specially in larger PRs.

Reply:
These were really unexpected change, probably I did a '%s/Expected/Expect/g`

Reply:
Reverted this part

Reply:
Thanks